src/pages/interesting.js

```diff
@@ -18,41 +19,38 @@ const styles = theme => ({
         marginRight: "auto",
         marginLeft: "auto"
     }
-});
+}));
 
-class InterestingPage extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            articles: [],
-            dataLoaded: false
-        }
-    }
-    componentWillMount() {
-        this.getData();
-    }
-    async getData() {
-        let articles = await axios.get("http://localhost:3001/articles");
-        this.setState({ articles: articles, dataLoaded: true });
+function InterestingPage(props) {
+    const loadData = props.loadData;
+    useEffect((props) => {
+        const links = ["http://localhost:3001/articles", "http://localhost:3001/authors"]
+        loadData(links);
```

Comment:
you are already fetching this data in categories page. First check if data is present in store and if not then you should fetch.